README.md

```diff
@@ -1,2 +1,24 @@
 # wcd2reshare
```

Comment:
I would add instructions for running and verifying this locally in Docker, like we have in the timdex-pipeline-lambdas app. Some context: the reason we don't have those instructions in alma-webhook-lambdas is because that lambda actually calls other AWS services in addition to returning a value, so it would require a full docker-compose setup with those other services to run locally. Since this app is just returning a value, we can run the function and check the return value from it in Docker. I would note in the README, however, that the _function url_ cannot be tested locally and has to be verified in AWS dev.